pyproject.toml

```
@@ -60,7 +60,7 @@ build-backend = "setuptools.build_meta"
 
 [project.optional-dependencies]
 soap-bpnn = [
-    "rascaline-torch @ git+https://github.com/luthaf/rascaline@5326b6e#subdirectory=python/rascaline-torch",
+    "featomic-torch",
```

Maybe pin the version here to a minor release

```suggestion
    "featomic-torch >=0.6,<0.7",
```